app/src/main/java/com/Found404/paypro/ui/pages/CardPaymentsPage.kt

```diff
@@ -0,0 +1,118 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun CardPayments(
+    onButtonFinishClick: () -> Unit
+) {
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .padding()
+            .background(color = Color.White),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceAround,
+    ) {
+        Text(
+            text = "Select card payments that you accept",
+            fontSize = 50.sp,
+            lineHeight = 50.sp,
+            fontWeight = FontWeight.Bold,
+            style = MaterialTheme.typography.titleLarge,
+            modifier = Modifier.padding(
+                vertical = 100.dp,
+                horizontal = 20.dp
+            )
+        )
+
+        Column() {
+            val checkBoxVisa by remember { mutableStateOf(false) }
+            val checkBoxMaster by remember { mutableStateOf(false) }
+            val checkBoxMaestro by remember { mutableStateOf(false) }
+            val checkBoxDiners by remember { mutableStateOf(false) }
+            val checkBoxAmerican by remember { mutableStateOf(false) }
+
+            CreateRow(cardNameParam = "Visa", cardTypeParam = checkBoxVisa)
+            CreateRow(cardNameParam = "MasterCard", cardTypeParam = checkBoxMaster)
+            CreateRow(cardNameParam = "Maestro", cardTypeParam = checkBoxMaestro)
+            CreateRow(cardNameParam = "Diners Club", cardTypeParam = checkBoxDiners)
+            CreateRow(cardNameParam = "American Express", cardTypeParam = checkBoxAmerican)
+            //make a for loop instead of hard calling the function every time
+            //will do once database is live with all the card names
+        }
+
+        Box(modifier = Modifier.fillMaxSize()){
+            Button(
+                modifier = Modifier
+                    .align(Alignment.BottomEnd)
+                    .padding(
+                        horizontal = 20.dp,
+                        vertical = 20.dp
+                    )
+                    .size(
+                        width = 130.dp,
+                        height = 60.dp
+                    ),
+                onClick = {
+                    /*TODO*/
+                },
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = Color.Blue
+                )
+            ) {
+                Text(text = "Finish",
+                    color = Color.White,
+                    fontSize = 20.sp,
+                    fontWeight = FontWeight.Bold
+                )
+            }
+        }
+    }
+}
+
+@Composable
```

Comment:
Its great that you extracted the row in a function. But it would be even better if it was in a separate file. We're gonna need it on other places too